tests/test_benchmark.py

```diff
@@ -455,18 +454,11 @@ def test_hif2a():
     benchmark_hif2a(verbose=True, num_batches=2, steps_per_batch=100)
 
 
-def get_nonbonded_pot_params(bps: List[BoundPotential]) -> Optional[Tuple[BoundPotential[Nonbonded], NDArray]]:
-    for bp in bps:
-        if isinstance(bp.potential, SummedPotential):
-            for pot, params in zip(bp.potential.potentials, bp.potential.params_init):
-                if isinstance(pot, Nonbonded):
-                    return pot, params
-    return None, None
-
-
 def test_nonbonded_interaction_group_potential(hi2fa_test_frames):
     bps, frames, boxes, ligand_idxs = hi2fa_test_frames
-    nonbonded_potential, nonbonded_params = get_nonbonded_pot_params(bps)
+    nonbonded_potential, nonbonded_params = next(
+        (bp.potential, bp.params) for bp in bps if isinstance(bp.potential, Nonbonded)
+    )
```

Comment:
nit: This still could be a function to hide the complexity

Reply:
Changed in https://github.com/proteneer/timemachine/pull/1087/commits/c4309ed4b843f9201b16e8d71c3e712af7c16893